src/components/HomePage/FAQSection.js

```diff
@@ -124,7 +124,7 @@ function FAQSection() {
 			answer:
 				<>
 					We recommend attending some of our events or asking us questions in
-					the <Link href="https://discord.gg/eWmzKsY">ACM Discord</Link>. Check
+					the <Link href="https://discord.gg/3GSPECbCnE">ACM Hack Discord</Link>. Check
```

Comment:
Is this discord link a permanent discord invite link?

Reply:
Yes. this should be a permanent invite link.